script.js

```diff
@@ -27,59 +30,63 @@ const createIncrementingIdGetter = () => {
 const getTaskId = createIncrementingIdGetter();
 
 //Функции счетчики
-const count = {
+const countTasksByStatus = {
   //Функция для подсчета активных задач
-  active: arr => {
-    return arr.reduce((count, el) => {
-      return !el.completed ? count + 1 : count;
+  active: tasks => {
+    return Array.from(tasks.values()).reduce((count, task) => {
+      return !task.completed ? count + 1 : count;
     }, 0);
   },
 
   //Функция для подсчета выполненных задач
-  completed: arr => {
-    return arr.reduce((count, el) => {
-      return el.completed ? count + 1 : count;
+  completed: tasks => {
+    return Array.from(tasks.values()).reduce((count, task) => {
+      return task.completed ? count + 1 : count;
     }, 0);
   },
 };
 
 //Функция для обновления значений counter невыполненных задач
 const renderCounter = () => {
-  let activeTasksNumber = count.active(tasksList);
+  let activeTasksNumber = countTasksByStatus.active(store.list);
 
-  counter.textContent = `${activeTasksNumber} ${activeTasksNumber === 1 ? 'item' : 'items'} left!`;
+  nodes.counter.textContent = `${activeTasksNumber} ${
+    activeTasksNumber === 1 ? 'item' : 'items'
+  } left!`;
 };
 
 //Функция для отслеживания изменение состояние чекбоксов
-const checkboxHandler = taskId => event => {
+const checkboxHandler = taskID => event => {
```

Comment:
```suggestion
const checkboxHandler = taskId => event => {
```

я бы айди кэмелкейсом писал